src/OrchardCore.Modules/OrchardCore.Users/Controllers/AccountController.cs

```diff
@@ -94,6 +96,8 @@ public class AccountController : AccountBaseController
             _loginFormDisplayManager = loginFormDisplayManager;
             _updateModelAccessor = updateModelAccessor;
             _externalLoginHandlers = externalLoginHandlers;
+            // Reverse the order of services to prioritize external services first, placing them before the default implementation.
```

Comment:
"External" is used for the login all over the module with a different meaning, so rather:

```suggestion
            // Reverse the order of services to prioritize third-party implementations first, placing them before the default one.
```